shoop/core/models/configuration.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+
+from _base import ShoopModel
+
+
+@python_2_unicode_compatible
+class Configuration(ShoopModel):
```

Comment:
This should be ConfigurationItem IMHO, since Configuration is a collection of ConfigurationItems.